mocha/index.d.ts

```diff
@@ -110,6 +106,24 @@ declare class Mocha {
 
 // merge the Mocha class declaration with a module
 declare namespace Mocha {
+    interface ISuiteCallbackContext {
```

Comment:
I just used the convention used in the entire file. I can change it all let existing applications break that rely on the interface type names.